### PR TITLE
Remove opset8 from compatibility ngraph python API

### PR DESCRIPTION
**Pull request by artkuli**
### Tickets:
 - *68456*

### CODE REVIEW

**Review by jiwaszki — APPROVED** [review]
LGTM 🚀

File: runtime/bindings/python/tests_compatibility/test_ngraph/test_pooling.py [hunk 1]
```diff
@@ -125,33 +120,17 @@ def test_max_pool_strides():
...
-        dilations,
-        pads_begin,
-        pads_end,
-        kernel_shape,
-        rounding_type,
-        auto_pad,
-        index_et,
-    )
-    comp = rt.computation(maxpool_node, data_node)
+    avgpool_node = ng.max_pool(data_node, strides, pads_begin, pads_end, kernel_shape)
```
Comment (tomdol) [inline runtime/bindings/python/tests_compatibility/test_ngraph/test_pooling.py:128]:
```suggestion
    maxpool_node = ng.max_pool(data_node, strides, pads_begin, pads_end, kernel_shape)
```

**Review by tomdol — APPROVED** [review]
There are some leftovers that you can additionally change: `avgpool_node` -> `maxpool_node` in one of the test files. Apart from that LGTM.

**Comment by ilyachur** [comment]
@tomdol  @jiwaszki Why did we remove opset8 from legacy python?

In C++ we have new opsets and operations in both APIs.

**Pull request merged**